src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/GetAzureWebAppSlot.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.Commands.WebApps.Cmdlets.DeploymentSlots
     /// this commandlet will let you get a new Azure Web app slot using ARM APIs
     /// </summary>
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "WebAppSlot")]
-    [OutputType(typeof(Site))]
+    [OutputType(typeof(PSSite))]
```

Comment:
Looks like you didn't change the actual output here either - WebsitesClient.ListWebApps and WebsitesClient.GetWebApp should be wrapped in PSSite

Reply:
Done